pyfr/solvers/base/system.py

```diff
@@ -300,3 +313,16 @@ def filt(self, uinoutbank):
 
     def ele_scal_upts(self, idx):
         return [eb[idx].get() for eb in self.ele_banks]
+
+    def get_ele_entmin_int(self):
+        if all(self.eles_entmin_int):
```

Comment:
EAFP.

Reply:
Try/except or just let it throw an error?

Reply:
I'd probably just let it throw, the error message you'll get by default will be about an undefined attribute, which is useful enough on its own.  Should be able to figure out from this that querying this variable only makes sense when EF is enabled.